cpp/ql/lib/semmle/code/cpp/security/InvalidPointerDereference/AllocationToInvalidPointer.qll

```diff
@@ -124,11 +127,11 @@ module SizeBarrier {
     )
   }
 
-  private predicate operandGuardChecks(
-    IRGuardCondition g, Operand left, Operand right, int state, boolean edge
-  ) {
+  /**
+   * Holds if `small < large + state` holds if `g` evaluates to `edge`.
```

Comment:
`small` and `large` are not mentioned in the predicate itself. Should this be sync'ed up somehow?